src/README.md

```diff
@@ -13,15 +13,15 @@ actions:
     icon: stopwatch
 
   - text: " Install"
-    link: /get-started/install/index.html
+    link: /get-started/index.html
```

Comment:
Just checking - does this not get converted if you use `get-started/index.md` instead of relying on the future output URL format?

Reply:
That is what it should be and does work, but the link checker complains about it for some reason. This is the only file that it cares about. but Ill run a new check now that I have updated the lychee version.

Reply:
confirmed the vuepress routing handler manages `.md` or no file ending just fine but the link checker wants it to be `.html`. BUUUUT this is supposed to be `install.hml` so I will fix that.